nft/nfts.yaml

```diff
@@ -266,7 +267,11 @@ paths:
                 contractAddresses:
                   type: array
                   description: list of contract addresses to batch metadata requests for
-                  default: ['0xe785E82358879F061BC3dcAC6f0444462D4b5330', '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d']
+                  default:
+                    [
+                      '0xe785E82358879F061BC3dcAC6f0444462D4b5330',
+                      '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d',
+                    ]
```

Comment:
@ayang-rkjs is this a required param? if so can we mark it as such?

Info on how to do this here: https://stackoverflow.com/questions/40113049/how-to-specify-if-a-field-is-optional-or-required-in-openapi-swagger

Reply:
nope neither of these are required!